python/orca/src/bigdl/orca/data/file.py

```diff
@@ -537,3 +537,31 @@ def get_remote_files_with_prefix_to_local(remote_path_prefix, local_dir):
         except Exception as e:
             invalidOperationError(False, str(e), cause=e)
     return os.path.join(local_dir, prefix)
+
+
```

Comment:
Add some comment to tell the difference between the one in dllib and the one here.
For example, this decorator can load on each worker while that can only load on driver

Reply:
Added.